.travis.yml

```diff
@@ -12,23 +10,33 @@ matrix:
         - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
             bash -x tools/lint-pr-commit-message.sh ${TRAVIS_PULL_REQUEST};
           fi
+
+
     - name: "Linter"
       language: node_js
       node_js: "node"
       env:
         - NODE=$(which node)
+      install:
+        - make lint-py-build || true
```

Comment:
Is there a missing change to run `lint-py` (the `lint` target doesn't)?

Reply:
~I think the `lint` target will run it if the python dependencies are setup.~ no just `lint-ci`